dask/array/tests/test_random.py

```diff
@@ -39,16 +55,6 @@ def test_kwargs():
     assert 8 < x.mean() < 12
 
 
-def test_kwargs_size_or_shape():
-    a = normal(loc=10.0, scale=0.1, shape=(10, 10), chunks=(5, 5))
-    b = normal(loc=10.0, scale=0.1, size=(10, 10), chunks=(5, 5))
-    assert isinstance(a, Array)
-    assert isinstance(b, Array)
-    assert a.chunks == b.chunks
-
-    assert np.array(a).shape == np.array(b).shape
-
```

Comment:
ah indeed, I missed it. I think being consistent with numpy API is good. However I also think that numpy should deprecated `size=` in favor of `shape=` :).